src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/opentelemetry-tutorials/opentelemetry-tutorial-java.mdx

```diff
@@ -0,0 +1,1132 @@
+
+1. Clone the demo app and navigate to the following directory:
+    ```
+    git clone git@github.com:newrelic/newrelic-opentelemetry-examples.git
+    cd newrelic-opentelemetry-examples/getting-started-guides/java/Instrumented
+    ```
+2. In your terminal, set these two environment variables to send data to your New Relic account:
+    ```
+    export OTEL_EXPORTER_OTLP_HEADERS=api-key=INSERT_YOUR_NEW_RELIC_LICENSE_KEY
+    export OTEL_EXPORTER_OTLP_ENDPOINT=https://otlp.nr-data.net:4317
+
+    ```
+    * Make sure to use your [ingest license key](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+    * If your account is based in the EU, set the endpoint to: https://otlp.eu01.nr-data.net:4317
+
+3. Set this environment variable to name the demo app:
+    ```
+    export OTEL_SERVICE_NAME=getting-started-java
+    ```
+4. In the `java/Instrumented/` directory, run the application:
+    ```
+    ./gradlew bootRun
+    ```
+5. Open a new terminal and exercise the application from the `Instrumented/java` directory by running this command:
+    ```
+    ./load-generator.sh
+    ```
+
+    <Callout variant="tip">
+      Alternatively, you can reach the endpoint in the browser at this URL: `http://localhost:8080/fibonacci?n=INSERT_A_VALUE`. Replace `INSERT_A_VALUE` with value from 1 to 90. To generate an error, insert an integer outside the valid range.
+    </Callout>
+
+6. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > APM & services**.
+7. Click your new entity (service) called `getting-started-java` and explore the UI. For more tips about what to look for in the UI, see [View your data in New Relic](#view-data).
+8. When you're finished looking at your data in the UI, shut down the application by pressing CONTROL+C in both terminal sessions.
+
+## Use the OpenTelemetry Java agent [#java-agent]
+
+OpenTelemetry provides an agent that automatically instruments your Java apps and doesn't require you to  modify your source code. The agent is a quick way for you to get started so you can get sample data into New Relic.
+
+The auto-instrumentation agent is a JAR file that dynamically injects bytecode to capture telemetry from popular libraries and frameworks. You can also use it to capture data such as inbound requests, outbound HTTP calls, and database calls. It can be attached to any Java 8+ application.
+
+<Callout variant="tip">
+  See the [official documentation](https://opentelemetry.io/docs/instrumentation/java/automatic/) for additional configuration options.
+</Callout>
+
+To monitor our demo app with the OpenTelemetry Java agent:
+
+1. Download the [agent JAR file](https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases/latest/download/opentelemetry-javaagent.jar), which contains the agent and instrumentation libraries. Place it in your preferred directory and note the path, which you will use later.
+2. If you haven't already done so, complete the following:
```

Comment:
might say what this step is doing. it's not clear to me if we expect them to first do the first tutorial and then the second one, or how often that's the case.

Reply:
@zuluecho9 I changed the intro to this section which might help differentiate it from the previous tutorial:

Here's a different tutorial that also uses the same demo app, but in this case, you'll use the OpenTelemetry Java agent to automatically monitor the demo app. You don't need to modify the Java source code. By using the agent, you can quickly start exporting sample data to New Relic.